app/keepers.go

```diff
@@ -367,16 +369,17 @@ func (app *NibiruApp) InitKeepers(
 		),
 	)
 
-	app.EvmKeeper = evmkeeper.NewKeeper(
+	evmKeeper := evmkeeper.NewKeeper(
 		appCodec,
 		keys[evm.StoreKey],
 		tkeys[evm.TransientKey],
 		authtypes.NewModuleAddress(govtypes.ModuleName),
 		app.AccountKeeper,
-		app.BankKeeper,
+		nibiruBankKeeper,
```

Comment:
Written this way to make sure the EVM keeper and its constructor have the stronger typed `NibiruBankKeeper` rather than an a `bankkeeper.Keeper` interface